### ISSUE TITLE
Leases sometimes get stuck in a loop after the lease expires

### ISSUE THREAD

**Issue by connor4312** [mixer/etcd3#50]
I've seen this happen twice in in the past several months:

```
Uncaught error: EtcdError: etcdserver: requested lease not found
    at /Users/copeet/Github/chat/node_modules/grpc/src/node/src/client.js:554:15
Uncaught error: EtcdError: etcdserver: requested lease not found
    at /Users/copeet/Github/chat/node_modules/grpc/src/node/src/client.js:554:15
Uncaught error: EtcdError: etcdserver: requested lease not found
    at /Users/copeet/Github/chat/node_modules/grpc/src/node/src/client.js:554:15
```

...forever

**Comment by connor4312** [comment]
Fixed in https://github.com/mixer/etcd3/pull/53

**Comment by connor4312** [comment]
Published in 0.2.7
